features/steps/insights_results_aggregator.py

```diff
@@ -29,9 +29,16 @@ def run_insights_results_aggregator_with_flag(context, flag):
     start_aggregator(context, flag, environment)
 
 
+@when("I remove all Insights Results Aggregator environment variables")
+def remove_insights_results_aggregator_env_vars(context):
+    context.no_IRA_environment = {key: value for (key, value) in os.environ.copy().items() if
```

Comment:
it does not do what was written in @when :)

Reply:
True, not the best name for what it actually does.

"I store current environment without Insights Results Aggregator environment variables", maybe?

Reply:
updated